addons/resource.language.en_gb/resources/strings.po

```diff
@@ -8162,11 +8163,13 @@ msgctxt "#19109"
 msgid "Couldn't save timer.[CR]Check the log for more information about this message."
```

Comment:
could you please also add `#: xbmc/pvr/timers/PVRTimerInfoTag.cpp` to `msgctxt #19109`